sdk/eventhub/azure-eventhubs/azure/eventhub/aio/receiver_async.py

```diff
@@ -156,18 +165,46 @@ def __aiter__(self):
             self._handler = ReceiveClientAsync(
                 source,
                 auth=self.client.get_auth(**alt_creds),
-                debug=self.client.debug,
+                debug=self.client.config.network_tracing,
                 prefetch=self.prefetch,
                 link_properties=self.properties,
                 timeout=self.timeout,
                 error_policy=self.retry_policy,
                 keep_alive_interval=self.keep_alive,
                 client_name=self.name,
-                properties=self.client.create_properties(),
+                properties=self.client.create_properties(self.client.config.user_agent),
                 loop=self.loop)
-        await self._handler.open_async()
-        while not await self._handler.client_ready_async():
-            await asyncio.sleep(0.05)
+
+        if not self.running:
+            try:
+                await self._handler.open_async()
+                self.running = True
+                while not await self._handler.client_ready_async():
+                    await asyncio.sleep(0.05)
```

Comment:
Why are we awaiting 0.05s? And is there any (easy) way to avoid taking a hard dependency on the async stack being asyncio?

Reply:
awaiting 0.05s is from track one, follow-up issue(#5618 ) was created to understand why and see if we can change it. I think this is related to uamqp.
For removing asyncio dependency, we have another dependency in async receiver and sender. I created a separate issue (#5617 )
```python
self.loop = loop or asyncio.get_event_loop(). 
```